modules/lang-groovy/src/test/java/org/elasticsearch/messy/tests/SimpleSortTests.java

```diff
@@ -1650,8 +1650,8 @@ public void testNestedSort() throws IOException, InterruptedException, Execution
             assertThat(hits[i].getSortValues().length, is(1));
             Object o = hits[i].getSortValues()[0];
             assertThat(o, notNullValue());
-            assertThat(o instanceof StringAndBytesText, is(true));
-            StringAndBytesText text = (StringAndBytesText) o;
+            assertThat(o instanceof Text, is(true));
```

Comment:
I know you've just copied but is it worth even asserting this? Also, if you do assert this maybe use `assertThat(o, instanceOf(Text.class))`;